README.md

```diff
@@ -1,9 +1,12 @@
-# th2 Sailfish Utils (3.8.1)
+# th2 Sailfish Utils (3.9.0)
 
 This library contains classes to convert messages from th2 to Sailfish format and vice versa. They are used in several th2 projects to reuse Sailfish features: message comparison, codec/connect implementations, etc.
 
 ## Release Notes
 
+### 3.9.0
++ Update th2-common version for new Filter SimpleList usage
```

Comment:
Please, add information about the previous and the new versions of the common library